### ISSUE TITLE
Query Reviewer for Datamapper

### ISSUE THREAD

**Issue by arkiver** [nesquena/query_reviewer#28]
I am using MySql and Datamapper on a Rails 3.2.11 app. Was trying to use query reviewer but getting this error on server start: 

.rvm/gems/ruby-1.9.3-p194@bookings/gems/query_reviewer-0.1.8/lib/query_reviewer/rails.rb:6:in `inject_reviewer': uninitialized constant QueryReviewer::ActiveRecord (NameError)

I guess query reviewer does not support DataMapper yet ? Let me know if you would prefer a full stack trace.

**Comment by arkiver** [comment]
If datamapper is not supported,  with some guidance, I will try to add code to support datamapper as well.

**Comment by nesquena** [comment]
You are right currently query reviewer has certain expectations of active record. I'd love for a future version to support any activemodel compliant orm. If you are interested in helping add datamapper support I suspect it wouldn't be too difficult. 
—
Nathan Esquenazi
CodePath Co-founder
http://thecodepath.com
